src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Requests/DraftApprenticeship/PostPriorLearningDataRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests.DraftApprenticeship
+{
+    public class PostPriorLearningDataRequest : IPostApiRequest
+    {
+        private readonly long _cohortId;
+        private readonly long _draftApprenticeshipId;
+        public object Data { get; set; }
+
+        public PostPriorLearningDataRequest(long cohortId, long draftApprenticeshipId)
+        {
+            _cohortId = cohortId;
+            _draftApprenticeshipId = draftApprenticeshipId;
+        }
+        public string PostUrl => $"cohorts/{_cohortId}/draft-apprenticeships/{_draftApprenticeshipId}/prior-learning-data";
```

Comment:
It would be better if the route pattern that we used when doing the delivery model work was adopted. In this pattern, the route in the outer api matches that in the UI, except that all routes are prefixed `provider/{providerId}` or `employer/{employerId}` and parameters are raw values rather than encoded. There are a number of endpoints that follow that pattern as an example, but I can send you more info.

Reply:
Gotcha, have changed this to:

provider/{_providerId}/unapproved/{_cohortId}/apprentices/{_draftApprenticeshipId}/edit/prior-learning-data